common/config/rush/common-versions.json

```diff
@@ -83,23 +83,29 @@
       "~0.6.1" // API Extractor is using an older version of source-map because newer versions are async
     ],
 
-    "webpack": ["~5.35.1"],
+    "webpack": ["~4.44.2"],
 
     // Use two different versions of @types/webpack-dev-server to allow pnpmfile.js to bring in
     // two different versions of the webpack typings
     "@types/webpack-dev-server": ["3.11.3"],
 
-    "webpack-dev-server": [
-      "~3.11.0" // heft-plugins/heft-webpack4-plugin is using an older version of webpack-dev-server for compatibility
-    ],
-
-    "html-webpack-plugin": [
-      "~4.5.0" // Some test projects are still on Webpack 4 because some of our plugins don't support Webpack 5 yet
-    ],
+    // heft-plugins/heft-webpack4-plugin is using an older version of webpack-dev-server for compatibility
+    "webpack-dev-server": ["~3.11.0"],
 
     "tapable": [
       "2.2.1",
       "1.1.3" // heft plugin is using an older version of tapable
-    ]
+    ],
+
+    // --- For Webpack 4 projects ----
+    "css-loader": ["~5.2.7"],
+    "html-webpack-plugin": ["~4.5.2"],
+    "postcss-loader": ["~4.1.0"],
+    "sass": ["~1.3.0"],
+    "sass-loader": ["~10.0.0"],
+    "source-map-loader": ["~1.1.3"],
+    "style-loader": ["~2.0.0"],
+    "terser": ["~4.8.0"],
+    "terser-webpack-plugin": ["~3.0.8"]
```

Comment:
```suggestion
    "css-loader": ["~5.2.7"],
    "html-webpack-plugin": ["~4.5.2"],
    "postcss-loader": ["~4.1.0"],
    "sass-loader": ["~10.0.0"],
    "sass": ["~1.3.0"],
    "source-map-loader": ["~1.1.3"],
    "style-loader": ["~2.0.0"],
    "terser-webpack-plugin": ["~3.0.8"],
    "terser": ["~4.8.0"]
```

Reply:
Default sort in JS, which is what we would normally use if generating this programmatically, begs to differ:
```
['test', 'test-foo', 'test'].sort();
-> ['test', 'test', 'test-foo']
```

Reply:
Friends, if we're going to have strong opinions about sorting every list, let's implement a feature to automate that. 😋 

I do try to sort the lists when I'm working on them, but we want to keep the code review discussion focused on the actual code.